scripts/delay_filter_run.py

```diff
@@ -4,6 +4,7 @@
 # Licensed under the MIT License
 
 """Command-line drive script for hera_cal.delay_filter"""
+"""Only performs CLEAN Filtering"""
```

Comment:
don't you mean to use `delay_clean_argparser`?

Reply:
Yep! Thanks for catching that.